docs/cvl/builtin.md

```diff
@@ -1,10 +1,27 @@
 Built-in Rules
 ==============
+`Certora` Prover has built-in general-purpose rules targeted at finding known vulnerabilities.
+These rules can be verified on a contract out-of-the-box.
 
-The Prover has some built-in general-purpose rules that can be verified on a
-contract out-of-the-box.
+You can add several builtin rules and all other rule types in the same spec file.
```

Comment:
are we sure about this? did we ever test it?

Reply:
I tested part of the features in a spec file. I assume it was checked for the hasdelegatecalls which was written first.